ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Selection-Tool/Lasso-Selection-Tool/lasso-tool.spec.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
I propose to not to add such comments, otherwise it makes no sense to use eslint rules, if we just disable them everywhere